xml/admin_monitoring.xml

```diff
@@ -94,4 +96,69 @@ https://raw.githubusercontent.com/kubernetes/heapster/release-1.3/deploy/kube-co
 <screen>&prompt.root;<command>kubectl apply -f grafana-service.yaml</command></screen>
   </sect2>
  </sect1>
+
+ <sect1 xml:id="sec.admin.monitoring.cluster.health">
+  <title>Cluster Health Checking</title>
+  <para>
+   Although &kube; takes care of a lot of traditional deployment problems
+   automatically, it is considered good practice to monitor the availability
```

Comment:
Maybe this is a good chance to state the Kubernetes philosophy here: that it's self-healing.